internal/pkg/cli/pipeline_delete.go

```diff
@@ -176,21 +179,34 @@ func (o *deletePipelineOpts) askAppName() error {
 }
 
 func (o *deletePipelineOpts) getSecret() error {
-	path, err := o.ws.PipelineManifestLegacyPath()
-	if err != nil {
-		return fmt.Errorf("get path to pipeline manifest: %w", err)
+	// Look for default secret name for GHv1 access token based on default pipeline name.
+	o.ghAccessTokenSecretName = o.pipelineSecretName()
+	output, err := o.secretsmanager.DescribeSecret(o.ghAccessTokenSecretName)
+	if err == nil {
+		// Check for Copilot-assigned tag for added assurance. If tags not found, fall through.
+		for _, tag := range output.Tags {
+			if aws.StringValue(tag.Key) == "copilot-application" && aws.StringValue(tag.Value) == output.CreatedDate.UTC().Format(time.UnixDate) {
```

Comment:
Maybe we should use `deploy.AppTagKey`?

Reply:
Oh cool, thanks. I didn't know that existed! 😆